pkg/sdk/external_tables_integration_test.go

```diff
@@ -0,0 +1,392 @@
+package sdk
```

Comment:
As I see, some of your tests are not passing. Did you try to running the same queries by hand? Did you try to adjust them according to the SQL error suggestions?

For tag, try with simpler name, I guess sometimes the strangely generated names are not working (not for all resources).
Getting tag in one of the tests is also suggesting to use slightly different syntax.

Reply:
Yes, 3 tests are failing, because although I think queries are correct according to the Snowflake documentation they're not correct, because they're failing to run (In the test and by hand). I'll try to run them with different names(ids) or variants of syntax for tags to see if anything will work.

Reply:
Well, Delta table is passing, but Set/Unset Tags are still not passing. This is something different and weird, because I believe it says "tags" after "set" is "Syntax Error" while in the documentation "tags" keyword is clearly after "set"